### PR TITLE
chore(ci): remove guard-component-size CI job

### PR DESCRIPTION
**Pull request by chasestarr**
This job has been replaced by [bundle-size](https://github.com/uber/baseweb/blob/master/.buildkite/pipeline.yml#L174) job

### CODE REVIEW

File: src/snackbar/snackbar-context.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ export default function SnackbarProvider({
 
   function enqueue(elementProps, duration = defaultDuration) {
     setSnackbars(prev => {
+      console.log('enqueue');
```
Comment (williamernest) [inline src/snackbar/snackbar-context.js:72]: Probably doesn't belong here.

Comment (chasestarr) [reply src/snackbar/snackbar-context.js:72]:
```suggestion
```

File: src/snackbar/snackbar-context.js [hunk 2]
```diff
@@ -93,10 +93,11 @@ export default function SnackbarProvider({
   }
 
   function enter(duration) {
+    console.log('enter');
```
Comment (chasestarr) [inline src/snackbar/snackbar-context.js:96]:
```suggestion
```

File: src/snackbar/snackbar-context.js [hunk 3]
```diff
@@ -95,8 +95,8 @@ export default function SnackbarProvider({
   function enter(duration) {
     setAnimating(true);
     setTimeout(() => {
-      setAnimating(false);
       display(duration);
+      setAnimating(false);
```
Comment (williamernest) [inline src/snackbar/snackbar-context.js:99]: Is this intentional?

**Review by williamernest — APPROVED** [review]

**Pull request merged**